olive/model/handler/onnx.py

```diff
@@ -116,6 +118,14 @@ def prepare_session(
         except OrtSessionFallbackError as e:
             raise OliveEvaluationError(e) from e
 
+    def run_session(
+        self,
+        session: Any = None,
+        inputs: Union[Dict[str, Any], List[Any], Tuple[Any, ...]] = None,
```

Comment:
can you generalize this to just `*args` and `**kwargs`? `InferenceSession.run` has this signature https://github.com/microsoft/onnxruntime/blob/8448f31d9042b6129886fb6cebfb23d6fe3293b8/onnxruntime/python/onnxruntime_inference_collection.py#L202 which is non standard

Reply:
I am not sure if that's good idea for this function. The implementation in different handlers is already specializing (in some cases, calling session(...) and others using explicit session.run(....)). Hiding the arguments behind args will make the function even more ambiguous and opaque. Having `inputs` as explicit input is probably better.

Will make the proposed change if you feel strongly that generalizing is a good idea.

Reply:
As discussed offline, updated the OrtInferenceSession::run args and OnnxModelHandler::run_session to account for the change.